src/Build.OM.UnitTests/Construction/ProjectItemElement_Tests.cs

```diff
@@ -1038,4 +1038,4 @@ private static ProjectItemElement GetItemFromContent(string content)
             return Helpers.GetFirst(project.Items);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
SA 1518 even applies to newlines at the end of files? I'm not (personally) opposed to taking those out, but I imagine they'd come back in with various editors so probably not worth worrying about. Removing multiple blank lines at the end of a file seems more clear-cut to me.

Reply:
The analyser uses the editor config settings for new lines. So I don't think they will get overridden by the editor later unless the settings change.